<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmallVariantSearchWidget</class>
 <widget class="QWidget" name="SmallVariantSearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This dialog is used to search for rare variants in the NGSD. &lt;br/&gt;The output can be copied to the clipboard and used in Excel or similar programs.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Variant</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <property name="spacing">
       <number>3</number>
      </property>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <widget class="QDoubleSpinBox" name="filter_af">
          <property name="suffix">
           <string>%</string>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>658</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="5" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <widget class="QCheckBox" name="filter_impact_high">
          <property name="text">
           <string>high</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="filter_impact_moderate">
          <property name="text">
           <string>moderate</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="filter_impact_low">
          <property name="text">
           <string>low</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="filter_impact_modifier">
          <property name="text">
           <string>modifier</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="region">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>chr:start-end</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radio_region">
        <property name="toolTipDuration">
         <number>-40</number>
        </property>
        <property name="text">
         <string>search by coordinates:</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="toolTip">
         <string>Maximum allele frequency in 1000G/gnomAD.</string>
        </property>
        <property name="text">
         <string>max. frequency:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_2">
        <property name="toolTip">
         <string>Variant impact on protein.</string>
        </property>
        <property name="text">
         <string>impact:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>max. NGSD count:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBox" name="filter_recessive">
        <property name="text">
         <string>recessive (only hom/compound-het variants)</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>inheritance:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QCheckBox" name="filter_denovo">
        <property name="text">
         <string>de-novo (from report config)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QSpinBox" name="filter_ngsd_count">
          <property name="maximum">
           <number>99999</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>50</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>5</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="mosaic_as_het">
          <property name="text">
           <string>also count mosaic variants</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="radio_genes">
        <property name="text">
         <string>search by genes:</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="genes">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Uses the gene locus extended by 5000 bases as region.</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Gene names</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Processed sample</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <property name="spacing">
       <number>3</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>quality:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>processing system type:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <widget class="QCheckBox" name="filter_sys_wgs">
          <property name="toolTip">
           <string>Short-read whole genome sequencing.</string>
          </property>
          <property name="text">
           <string>WGS</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="filter_sys_lrgs">
          <property name="toolTip">
           <string>Long-read whole genome sequencing.</string>
          </property>
          <property name="text">
           <string>lrGS</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="filter_sys_wes">
          <property name="toolTip">
           <string>Short-read whole exome sequencing.</string>
          </property>
          <property name="text">
           <string>exomes</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="filter_sys_other">
          <property name="toolTip">
           <string>Panels, cfDNA, etc.</string>
          </property>
          <property name="text">
           <string>others</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <widget class="QCheckBox" name="q_ps_good">
          <property name="text">
           <string>good</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="q_ps_medium">
          <property name="text">
           <string>medium</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="q_ps_bad">
          <property name="text">
           <string>bad</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="q_ps_na">
          <property name="text">
           <string>n/a</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>project type:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <widget class="QCheckBox" name="p_diagnostic">
          <property name="text">
           <string>diagnostic</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="p_research">
          <property name="text">
           <string>research</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="p_external">
          <property name="text">
           <string>external</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="p_test">
          <property name="text">
           <string>test</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="update_btn">
       <property name="text">
        <string>search</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="copy_btn">
       <property name="toolTip">
        <string>Copy entire table to clipboard.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="GSvar.qrc">
         <normaloff>:/Icons/CopyClipboard.png</normaloff>:/Icons/CopyClipboard.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+S</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="variants">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="columnCount">
      <number>25</number>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>gene</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>variant</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>NGSD hom</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>NGSD het</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>NGSD mosaic</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>gnomAD AF</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>CADD</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>SpliceAI</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>variant type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>coding and splicing</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>processed sample</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>external sample name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>gender</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>genotype</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>processing system</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>project</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>disease_group</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>disease_status</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>HPO terms</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>classification</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>diagnostic outcome</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>diagnostic outcome comment</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>causal genes</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>candiate genes</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>related samples</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>radio_genes</tabstop>
  <tabstop>radio_region</tabstop>
  <tabstop>genes</tabstop>
  <tabstop>region</tabstop>
  <tabstop>filter_af</tabstop>
  <tabstop>filter_ngsd_count</tabstop>
  <tabstop>filter_impact_high</tabstop>
  <tabstop>filter_impact_moderate</tabstop>
  <tabstop>filter_impact_low</tabstop>
  <tabstop>filter_impact_modifier</tabstop>
  <tabstop>filter_recessive</tabstop>
  <tabstop>filter_denovo</tabstop>
  <tabstop>q_ps_good</tabstop>
  <tabstop>q_ps_medium</tabstop>
  <tabstop>q_ps_bad</tabstop>
  <tabstop>q_ps_na</tabstop>
  <tabstop>p_diagnostic</tabstop>
  <tabstop>p_research</tabstop>
  <tabstop>p_external</tabstop>
  <tabstop>p_test</tabstop>
  <tabstop>filter_sys_wgs</tabstop>
  <tabstop>filter_sys_wes</tabstop>
  <tabstop>filter_sys_other</tabstop>
  <tabstop>update_btn</tabstop>
  <tabstop>copy_btn</tabstop>
  <tabstop>variants</tabstop>
 </tabstops>
 <resources>
  <include location="GSvar.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
